src/Tracker/index.js

```diff
@@ -82,9 +97,15 @@ const formatProperties = properties => {
   )
 }
 
+const userAnalyticsEvent = (eventName, properties) =>
+  dispatchEvent(new CustomEvent('userAnalyticsEvent', {detail: {eventName, properties}}));
+
 const sendEvent = (eventName, properties) => {
-  if (shouldSendEvents)
+  if (shouldSendEvents) {
+    if (integratorTrackedEvents.includes(eventName))
```

Comment:
Minor: everywhere else in the code we enclose the consequent of an `if` statement into curly brackets, like so `if (condition) { consequent_logic }`. We will be adding stricter liner rules this sprint.